deploy/charts/rook-ceph/templates/deployment.yaml

```diff
@@ -265,6 +265,10 @@ spec:
         - name: CSI_CEPHFS_PLUGIN_RESOURCE
           value: {{ .Values.csi.csiCephFSPluginResource | quote }}
 {{- end }}
+{{- if .Values.monitoring.prometheusRule }}
+        - name: ROOK_CEPH_MONITORING_PROMETHEUS_RULE
```

Comment:
Where is set when not using Helm? `operator.yaml` needs some changes too.
Also, these rules will apply to all the cephcluster deployed right?